dist-git/conf/tmpfiles.d/copr-dist-git.conf

```diff
@@ -1 +1,2 @@
 D /var/lock/copr-dist-git 0700 copr-dist-git copr-dist-git -
+d /var/lib/copr-dist-git 0755 copr-dist-git copr-dist-git -
```

Comment:
```
[root@copr-dist-git ~][PROD]# rpm -qf /var/lib/copr-dist-git/
copr-dist-git-1.1-1.fc41.noarch
```

Commit message:
```
Also, the user home directory is not created automatically.
```

It is, because it is a [part of the copr-dist-git.rpm's payload](https://github.com/fedora-copr/copr/blob/62d0f7f8c8710b1cdd7bd0ba2d3761c8ab7f6ae8/dist-git/copr-dist-git.spec#L120)?